app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/FriendsList.kt

```diff
@@ -110,51 +111,42 @@ fun FriendsCard(friend: UserModel, navToFriend: () -> Unit) {
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.climbing_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.climbing_expert)
-                            }
-                        ActivityType.HIKING ->
-                            when (prefActivityLevel) {
-                              UserLevel.BEGINNER ->
-                                  LocalContext.current.getString(R.string.hiking_newcomer)
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.hiking_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.hiking_expert)
-                            }
-                        ActivityType.BIKING ->
-                            when (prefActivityLevel) {
-                              UserLevel.BEGINNER ->
-                                  LocalContext.current.getString(R.string.biking_newcomer)
-                              UserLevel.INTERMEDIATE ->
-                                  LocalContext.current.getString(R.string.biking_amateur)
-                              UserLevel.ADVANCED ->
-                                  LocalContext.current.getString(R.string.biking_expert)
-                            }
-                      })
-            }
+                }
           }
         }
+        Row(
+            modifier = Modifier.testTag("FriendPrefActivity").fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween) {
+              // displays the user's sport preference
+              Row(verticalAlignment = Alignment.CenterVertically) {
+                Text(
+                    modifier = Modifier.testTag("friendInfo").padding(5.dp),
+                    text = friend.descrText(),
+                    fontSize = 15.sp)
+                Image(
+                    painter = friend.prefActivity.getIcon(),
+                    contentDescription = "Preferred activity logo",
+                    modifier = Modifier.size(25.dp, 25.dp))
+              }
+              // displays the number of friends in common
+              val friendsInCommonText =
+                  when (friend.getCommonFriends(friendList)) {
+                    0 -> LocalContext.current.getString(R.string.no_friends_in_common)
+                    1 -> LocalContext.current.getString(R.string.one_friends_in_common)
+                    else ->
+                        "${friend.getCommonFriends(friendList)} ${LocalContext.current.getString(R.string.friends_in_common)}"
+                  }
+              Row { Text(text = friendsInCommonText, fontSize = 12.sp, textAlign = TextAlign.End) }

```

Comment:
Here the fontSize shouldn't be hardcode! Try to use the MaterialTheme.typography package where you can find many font styles!